components/places/src/import/fennec/history.rs

```diff
@@ -13,8 +13,9 @@ use sql_support::ConnExt;
 use std::time::Instant;
 use url::Url;
 
-// From https://searchfox.org/mozilla-central/rev/597a69c70a5cce6f42f159eb54ad1ef6745f5432/mobile/android/base/java/org/mozilla/gecko/db/BrowserDatabaseHelper.java#73.
-const FENNEC_DB_VERSION: i64 = 39;
+// Fennec's history schema didn't meaningfully change since 34, so this could go as low as that version.
+// However, 36 was quite easy to obtain test databases for, and it shipped with quite an old ESR version (52).
+const FENNEC_DB_VERSION: i64 = 34;
```

Comment:
The comment sounds like 36 is used but 34 is used. Is this intended?

Reply:
Ah, good catch - comment is wrong here, I switched it around a few times.